pages/admin/AdminHandler.php

```diff
@@ -119,11 +119,11 @@ public function initialize($request)
             ]);
         }
 
-        // Interact with the beacon (if enabled) and determine if a new version exists
-        $latestVersion = VersionCheck::checkIfNewVersionExists();
-
         // Display a warning message if there is a new application version available
         if (Config::getVar('general', 'show_upgrade_warning') && $latestVersion) {
+            // Interact with the beacon (if enabled) and determine if a new version exists
+            $latestVersion = VersionCheck::checkIfNewVersionExists();
```

Comment:
This change will result in the version check never being run, and a warning, because `$latestVersion` (one line above) will never be defined.

Reply:
Sorry about that, fixed it.